src/parameterizations/lateral/MOM_mixed_layer_restrat.F90

```diff
@@ -338,7 +349,21 @@ subroutine mixedlayer_restrat_OM4(h, uhtr, vhtr, tv, forces, dt, h_MLD, VarMix,
   h_neglect = GV%H_subroundoff
   if (CS%front_length>0.) then
     res_upscale = .true.
-    I_LFront = 1. / CS%front_length
+    do j=js-1,je+1 ; do i=is-1,ie+1
+      mle_fl_2d(i,j) = CS%front_length
```

Comment:
This should probably be changed as follows, but this is not the only issue. I am leaving another comment below

```
mle_fl_2d(i,j) = CS%front_length * US%L_to_m
```